src/modules/app/onBeforeChange.js

```diff
@@ -10,21 +10,18 @@ import { getRouteConfig } from './route-configs'
  * This handles the loading of additional data for the location layout
  * (onBeforeChange is executed before a change of the route)
  */
-const onBeforeChange = async (dispatch: Dispatch, getState: GetState, bag: Bag): Promise<Payload<*>> => {
+const onBeforeChange = (dispatch: Dispatch, getState: GetState, bag: Bag): void => {
```

Comment:
Hm, if this does not wait on Promises we have a problem here.

Reply:
What problem?

Reply:
Promises should always be awaited, otherwise they can lead to bad side effects

Reply:
Also -- what was the reason, that these fetch operations are not part of the routes' thunks?

Reply:
I don't know anymore why they are here. Will try it out

Reply:
I think we should be fine by not waiting, but side effects are not clear.

Reply:
I already moved it to the route thunks, which also works. So we should be fine.